test/lib/util.js

```diff
@@ -596,13 +597,32 @@ async function waitForSometime(timeoutMS = 10 * second) {
   await new Promise(resolve => setTimeout(resolve, timeoutMS));
 }
 
+/**
+ * Get the regionalized RTCIceServers[].
+ * @param {string} token
+ * @param {string} region
+ * @param {object} options
+ * @returns {Promise<RTCIceServer[]>}
+ */
+async function getRegionalizedIceServers(token, region, options) {
```

Comment:
@manjeshbhargav - as part of my changes for in-band-turn we are planning to remove  `NTSIceServerSource`
If this test needs to get them off band, we should copy the `NTSIceServerSource` into test/lib/nts.js and use that version here.